docs/guides/getting-started/getting-started-scaling-and-tuning.adoc

```diff
@@ -0,0 +1,64 @@
+<#import "/templates/guide.adoc" as tmpl>
+<#import "/templates/links.adoc" as links>
+
+<@tmpl.guide
+title="Scaling"
+summary="Get started with {project_name} scaling and tuning">
+
+Once you have {project_name} started you can next consider adapting your instance to whatever load may be needed. Scaling and tuning can help you address:
+
+- minimizing resource utilization
+- excessive response times
+- database pool contention
+- out of memory errors, or excessive garbage collection overhead
+- providing higher availability via horizontal scaling
```

Comment:
This is mixing positive items (like "minimizing resource utilization") and negative items ("excessive response times"). To be better readable, I'd prefer those items to be all either positive or negative.

Reply:
Okay, see if the reprasing works for you.

Reply:
Thanks, looks now great to me.